### PR TITLE
feat: made add trip page

### PR DESCRIPTION
**Pull request by Takahiro9-Murakami9**

### CODE REVIEW

**Review by calss0t — COMMENTED** [review]
We have to be careful because YoonJu also made a pull request. I think it should be ok, but I would wait until one is approved and merged to approve the other

**Review by YJK-7 — APPROVED** [review]
looks like there shouldn't be any conflicts 👍

**Review by Raeki — APPROVED** [review]

**Pull request merged**
